.github/workflows/macos_python.yml

```
@@ -0,0 +1,60 @@
+# These are nightly package tests for macOS
+# focus areas:
+# - initial user experience
+# - scientific python stack
+name: macOS builds nightly
+
+on:
+  # TODO: remove PR
+  pull_request:
+  schedule:
+    # nightly at 1 AM
+    - cron: '0 1 * * *'
+
+# GitHub Action Limits
+# https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+jobs:
+  install_gcc:
+    name: gcc with clang
+    runs-on: macos-10.15
```

Should we use:
```suggestion
    runs-on: macos-latest
```
to avoid having to change the workflow in the future for a bump in the OS?

---

I wasn't sure but why not!